myapp/templates/single_stock_estimates.html

```diff
@@ -0,0 +1,36 @@
+{% extends "base.html" %}
+
+{% block content %}
+<div class="col-md-12 text-left">
+		<h2 class="display-4">
+				{{ CompanyName }}
+				<small class="text-muted">({{ symbol }})</small>
+		</h2>
+</div>
+<div class="row">
+	<div class="pt-4 col-sm-6 pl-5">
+		<dl class="row text-left">
```

Comment:
see how musab did this here: 
https://github.com/eladeck/StockProject/pull/11/files#diff-75b2a92124c50940fd49e9a60f9f2cea8ccf795d93a321afe1b28b6fcd6ddd6e

Reply:
Fixed.